docker-compose.yaml

```diff
@@ -484,6 +484,64 @@ services:
       permissionless-builder:
         condition: service_healthy
 
+  marketplace-reserve-builder:
+    image: ghcr.io/espressosystems/espresso-sequencer/marketplace-builder:main
```

Comment:
can we also add the docker file and CI to include the new binary?

Reply:
Sure! I added those, although I'm not that well-versed in docker, so mostly by analogue with existing files, hope it all makes sense.